lib/parsing/lexer.ml

```diff
@@ -53,11 +59,21 @@ let rec token lexbuf =
   | ':' -> COLON
   | '.' -> DOT
   | ',' -> COMMA
-  | '"' -> DOUBLE_QUOTE
+  (* | '"' -> DOUBLE_QUOTE *)
+  | '"' -> FILENAME (filename lexbuf)
```

Comment:
This would mean that every "string" should be treated as a filename, right? I'm fine with it for now (since we don't have strings 🤷 )

Reply:
Yes, for now every string is treated as a filename. I changed the name of the tokenizer function to reflect that, but actually the token itself should probably be just called `string` instead of `filename` (any kind of well-formedness checking on the filename should be done outside of the lexer/parser anyway)